unicodetools/src/test/java/org/unicode/test/TestSecurity.java

```diff
@@ -453,15 +453,17 @@ public void TestScriptDetection() {
         Set<Set<Script_Values>> expected = new HashSet<>();
         String[][] tests = {
             {"℮", "Common"},
+            {"1ℓ ℮", "Common"},
+            {"75 cl ℮", "Latin"},
```

Comment:
I got email notifications for comments on these from Roozbeh but don't see the comments here...??

Anyway, I don't mind adding a couple of new test strings.

Reply:
As mentioned in the PR description, the point is that they cover cases that were previously covered by the test cases with U+0300 (A pile of Common characters is Common, and a pile of common characters with some Latin is Latin).

Reply:
> I got email notifications for comments on these from Roozbeh but don't see the comments here...??

I deleted them. I saw @eggrobin's comment and understood the nature of the additional tests.